.vscode/template.launch.json

```diff
@@ -53,6 +53,15 @@
       "console": "integratedTerminal",
       "sourceMaps": true,
       "cwd": "${workspaceFolder}/examples/supply-chain-app/"
+    },
+    {
```

Comment:
@outSH Is there some documentation about how this should work? I tried it just now but it didn't appear to launc the VSCode debugger or any terminal processes so I couldn't figure out what it was doing.

Reply:
It only works when you already `docker-compose up` a sample app with debug yaml file, without that it does nothing. I want to add more utility for starting the sample apps in the future (after including them in monorepo etc...), didn't want to loose this config until then :)